playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```
@@ -35,6 +35,32 @@
     - name: umount ephemeral disk on ceph_osds
       shell: umount /dev/{{ item }}
       with_items: "{{ ceph.disks }}"
+      ignore_errors: true
```

This should be `failed_when: False` instead of `ignore_errors: true`.